engine/cld/changeset/registry_test.go

```diff
@@ -0,0 +1,340 @@
+package changeset
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+)
+
+// noopMigration is a migration that does nothing.
+//
+// Used for testing.
+type noopMigration struct {
```

Comment:
This could be renamed now ?
```suggestion
type noopChangeset struct {
```